packages/redis_client/README.md

```diff
@@ -7,6 +7,34 @@ A Dart library for interacting with [Redis](https://redis.io).
 [license_badge]: https://img.shields.io/badge/license-MIT-blue.svg
 [license_link]: https://opensource.org/licenses/MIT
 
+## Quick Start
+
+```dart
+import 'dart:async';
+
+import 'package:redis_client/redis_client.dart';
+
+Future<void> main() async {
+  // Create an instance of a RedisClient.
+  final client = RedisClient();
+
+  // Connect to the Redis server.
+  await client.connect();
```

Comment:
At what point is the server URL/other connection info specified?

Reply:
It's passed as `RedisSocketOptions` via the constructor but there are defaults preconfigured.